src/tests/testingLibraryHelpers.tsx

```diff
@@ -16,7 +16,7 @@ interface RenderOptions extends ReactTestingLibrary.RenderOptions {
  * container in their unit test** will have test failures on CI due to
  * serialization order being off.
  */
-const usePortalContainer = () => {
+const addPortalContainer = () => {
```

Comment:
`use*` syntax is reserved for the linter and so should be avoided when not really a hook